plugins/dialog/src/commands.rs

```diff
@@ -143,7 +143,7 @@ pub(crate) async fn open<R: Runtime>(
                     for folder in folders {
                         if let Ok(path) = folder.clone().into_path() {
                             if let Some(s) = window.try_fs_scope() {
-                                s.allow_directory(&path, options.recursive);
+                                let _ = s.allow_directory(&path, options.recursive);
```

Comment:
we should propagate the error, similar to how `tauri_scope` is propagated

Reply:
ah yes, of course